src/main_app/db/services/crud_service.py

```
@@ -37,18 +37,18 @@ def __init__(self, session: Session, model: type[ModelT]) -> None:
     # Read
     # ------------------------------------------------------------------ #
 
-    def get_record_by_id(self, pk: PKT) -> ModelT | None:
+    def get_record_by_id(self, pk: PKT) -> ModelT | None:  # pyright: ignore[reportInvalidTypeVarUse]
         """Fetch a single row by primary key, or None if it doesn't exist."""
         try:
             return self.session.get(self.model, pk)
         except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)